### ISSUE TITLE
Format field values not accessible if info field with same name exists

### ISSUE THREAD

**Issue by juliangehring** [brentp/cyvcf2#72]
If a VCF contains an INFO and a FORMAT field with the same name ('DQ' in the example)

```
[...]
##INFO=<ID=DQ,Number=1,Type=Integer>
##FORMAT=<ID=DQ,Number=1,Type=Float>
[...]
chr1	10000	.	C	G	.	LowGQX	DP=1277;MQ0=1179;DQ=1	GT:GQ:GQX:DP:FDP:AD:PL:DQ	0/0:54:54:18:1:18,0:0,54,370:0	0/0:90:90:30:2:30,0:0,90,370:.	0/1:12:12:11:6:10,1:12,0,300:.
```

retrieving the values of the format field return an empty object:

```python
vf = cyvcf2.VCF('path_to_file.vcf')
v = next(vf)
v.format('DQ')  # returns None
```

Without the INFO field present, it returns the values of the format field:

```python
v.format('DQ')
```

```
array([[  0.],
       [ nan],
       [ nan]], dtype=float32)
```

I'm not sure if the different types between the info and format field play a role here.

**Comment by brentp** [comment]
ouch. thanks for reporting. I have pushed a fix for this and sent cyvcf2 v0.8.4 to pypi.

**Comment by juliangehring** [comment]
Amazing, thanks for this super fast fix!
